SMARTMANUFACTURING/corosect/README.md

```diff
@@ -1,2 +1,13 @@
+# SUMMARY
```

Comment:
This will be moved to notes.yaml as long as README.md is overwritten by the automatic processes.